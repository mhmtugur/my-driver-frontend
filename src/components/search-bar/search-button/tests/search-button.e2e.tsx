import { E2EElement, E2EPage, newE2EPage } from '@stencil/core/testing';

describe('location list item', () => {
  it('should render button', async () => {
    let page: E2EPage = await newE2EPage({ html: '<search-button></search-button>' });
    let element: E2EElement = await page.find('search-button');
    expect(element.innerHTML).toMatchSnapshot();
  })
});