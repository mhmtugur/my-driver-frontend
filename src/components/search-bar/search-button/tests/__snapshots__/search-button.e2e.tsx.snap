// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`location list item should render button 1`] = `"<div class=\\"input-field\\"><button class=\\"btn waves-effect waves-light\\">Search</button></div>"`;
